.coveragerc

```diff
@@ -889,6 +889,7 @@ omit =
     homeassistant/components/pushover/notify.py
     homeassistant/components/pushsafer/notify.py
     homeassistant/components/pyload/sensor.py
+    homeassistant/components/p1_monitor/diagnostics.py
```

Comment:
Since the rest of the component is tested, I think it would make sense to add tests here also.

Reply:
I had some trouble writing the tests, but can give it another try.

Reply:
Tests are now added 😉